packages/fast-components-styles-msft/src/utilities/density.ts

```diff
@@ -170,8 +154,5 @@ export function glyphSizeNumber(designSystem: DesignSystem): number {
  * @param unit
  */
 export function density(value: number, unit?: string): (config: DesignSystem) => string {
-    return (config: DesignSystem): string => {
-        const augmented: number = value * 1;
-        return typeof unit === "string" ? `${augmented}${unit}` : toPx(augmented);
-    };
+    return (config: DesignSystem): string => toUnit(unit)(value * 1);
```

Comment:
is the `* 1` meant to convert any values to a number?

Reply:
No idea - just trying to keep the logic the same.